src/main/java/uk/gov/hmcts/reform/sscs/model/servicehearingvalues/ShvPanelRequirements.java

```diff
@@ -15,11 +15,12 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 @AllArgsConstructor
 @NoArgsConstructor
-public class PanelRequirements {
+public class ShvPanelRequirements {
 
     private List<String> roleType;
     private List<String> authorisationTypes;
     private List<String> authorisationSubType;
-    private List<PanelPreference> panelPreferences;
+    //@JsonProperty("SvhPanelPreferences")
```

Comment:
Json property needs to be panelPreferences

Reply:
Resolved